cylc/flow/resources.py

```diff
@@ -128,13 +128,13 @@ def _backup(tgt: Path) -> None:
     shutil.move(str(tgt), str(backup))
 
 
-def extract_resource(src: Path, tgt: Path) -> None:
+def extract_resource(src: Path, tgt: Path, is_tutorial: bool = False) -> None:
     """Extract src into tgt.
 
     NOTE: src can be a dir or a file.
     """
     LOG.info(f"Extracting {src.relative_to(RESOURCE_DIR)} to {tgt}")
-    if tgt.exists():
+    if tgt.exists() and is_tutorial:
```

Comment:
Probably better to skip this filesystem operation unless needed:
```suggestion
    if is_tutorial and tgt.exists():
```